### ISSUE TITLE
Error SSL en `https://es.python.org`

### ISSUE THREAD

**Issue by astrojuanlu** [python-spain/web-ng#93]
Todas las otras combinaciones redirigen bien:

- https://www.es.python.org ✔️ 
- http://es.python.org ✔️ 
- http://www.es.python.org ✔️ 

Pero https://es.python.org me da esto:

![Screenshot from 2021-11-11 10-45-56](https://user-images.githubusercontent.com/316517/141276106-100dc3f8-6a7d-43a2-825f-e2120f73af83.png)

**Comment by dukebody** [comment]
Arreglado, he puesto como custom domain `es.python.org` en vez de `www.es.python.org` y ya va todo bien.

**Comment by astrojuanlu** [comment]
Ojo que ahora falla https://www.es.python.org/ no? 🙈

**Comment by dukebody** [comment]
No, a mí me redirecciona sin problema y no da ningún error. ¿A ti te falla?

**Comment by quobit** [comment]
Falla https://www.es... pero http://www.es... redirige y va bien

**Comment by dukebody** [comment]
Comprobado que desde el móvil https://www.es.python.org da problemas. Esperemos que la gente no use la versión con www y ya.

**Comment by XaviTorello** [comment]
He recreado la config para volver a pasar todo el proceso y parece que ya funciona en todos los casos. En todos hace redirect a https://es.python.org

He probado con distintos dispositivos, varios proxies con distinta location (para probar entrando por distintos cdn).

El sistema de gh-pages y la capa TLS/SSL están cogidos con pinzas, no es la primera vez que me encuentro con poltergeists de este tipo...

Gracias @dukebody 

Si veis algo raro avisad y lo miramos. Según cómo, podríamos pasar a cloudfare (o similares) y ganar más control sobre el enrutamiento

**Comment by dukebody** [comment]
@XaviTorello desde el móvil https://www.es.python.org sigue dando problemas. En escritorio curiosamente no me da problemas.

Veo varias opciones:
1. Crear un repo para www y hacer redirección.
2. No apuntar el www a nada, y así quizá los navegadores lo intentan reescribir automáticamente. No debería haber muchos enlaces apuntando a la versión www.
3. Probar algo como Cloudflare Pages, Netlify... últimamente con los problemas de GH pages lo estoy viendo como posibles mejores opciones a simple vista, aunque nunca las he usado.
